examples/postgres-diskstats/loadfromscratch.sh

```diff
@@ -0,0 +1,10 @@
+# These commands are executed from outrigger container
+# to set up PostgreSQL table  and load data from a JSON file into that table,
+# using juttle commands with juttle-postgres-adapter.
+#
+# This assumes use of postgres_diskstats_data volume where incoming files are. 
+
+sleep 10  # wait for postgres to be ready
+juttle -e "read postgres -raw 'CREATE TABLE kb_out(name text,host text,time timestamp,value double precision);'"
```

Comment:
I believe instead of `timestamp` here it should be `timestamp with time zone` which preserves the timezone info.  Unfortunately simply `timestamp` type does not. http://www.postgresql.org/docs/9.2/static/datatype-datetime.html